statsmodels/duration/hazard_regression.py

```diff
@@ -1172,19 +1168,14 @@ def baseline_cumulative_hazard_function(self, params):
         return cumhaz_f
 
     def predict(self, params, exog=None, cov_params=None, endog=None,
-                strata=None, offset=None, pred_type="lhr"):
-        # docstring attached below
+                strata=None, offset=None, pred_type="lhr", 
+                return_object=False, **kwargs):
 
         pred_type = pred_type.lower()
-        if pred_type not in ["lhr", "hr", "surv", "cumhaz"]:
+        if pred_type not in ["lhr", "hr", "surv", "cumhaz", "qsurv"]:
```

Comment:
What I usually do is something like this:

First make sure you have pushed all your work to github

Then:

git checkout master
git reset --hard HEAD
git fetch upstream
git merge upstream/master
git push origin master -f
[login]
git checkout distributed estimation
git rebase master
[hopefully no merge conflicts]
git push origin distributed_estimation -f
[login]


On Thu, Jun 23, 2016 at 9:31 AM, Leland Bybee <notifications@github.com>
wrote:

> In statsmodels/duration/hazard_regression.py
> <https://github.com/statsmodels/statsmodels/pull/3055#discussion_r68234116>
> :
>
> >
> >          pred_type = pred_type.lower()
> > -        if pred_type not in ["lhr", "hr", "surv", "cumhaz"]:
> > +        if pred_type not in ["lhr", "hr", "surv", "cumhaz", "qsurv"]:
>
> Yeah, I do think they got crossed, I'm not sure what the best way is to
> fix that.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/statsmodels/statsmodels/pull/3055/files/04332954280820725c730b2764b96072a4603a32#r68234116>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACiwwwq1CbtKjpqeVoekLggvtkmUwgk7ks5qOoqagaJpZM4I6Q4M>
> .
>